src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimCallbackHandlerTest.java

```diff
@@ -483,7 +479,169 @@ void shouldValidateExperts_whenMultipartyAndSolicitorRepresentsOnlyOneOfResponde
 
             assertThat(response.getErrors()).isEmpty();
         }
+        @Test
+        void shouldValidateExperts_whenMultipartyAndSolicitorRepresentsOnlyOneOfRespondents() {
```

Comment:
How are you checking that the experts are being validated?

According to the assertion at the end, you're just checking that there are no errors but that doesn't guarantee that any validation has been carried out.

Let's check what part of code you're trying to validate and rework this test together.

Reply:
Done